tests/containers/helm.pm

```diff
@@ -28,15 +22,10 @@ sub run {
     $self->{deployment_name} = "apache-" . get_current_job_id();
     my $chart = "bitnami/apache";
 
-    if (is_sle) {
-        add_suseconnect_product(get_addon_fullname('phub'));
-        add_suseconnect_product(get_addon_fullname('pcm'));
-        zypper_call("in -y jq helm aws-cli");
-        install_kubectl();
-    }
-    else {
-        zypper_call("in -y jq aws-cli helm kubernetes-client");
-    }
+    add_suseconnect_product(get_addon_fullname('pcm')) if (is_sle);
+    install_kubectl();
+    install_helm();
+    zypper_call("in -y jq aws-cli");
```

Comment:
`jq` is already installed by `install_kubectl();`
But I would actually leave it here and remove it from `install_kubectl()` as it's not really a kubernetes package.